### ISSUE TITLE
Talk schedule should be on the first page

### ISSUE THREAD

**Issue by ViralBShah** [JuliaCon/juliacon.github.io#36]
I think the talk schedule should really be on the first page, as that is the driver of all interest. Any reason not to?

**Comment by ArchRobison** [comment]
I think the page would get a bit long.  I'd be inclined to shorten the page further by moving the program committee onto a separate page, since the program committee at this point is not a driver of interest.

The sentence "Submissions for talks are now closed, and the list of talks has been announced." hides the important link at the end.  Let's reword it to something more direct like "[List of talks and workshops.](http://juliacon.org/sched.html)".

**Comment by astrieanna** [comment]
I think a one-page website is a really good idea. I think having a long
page is fine (we can add anchor links if needed) -- it makes it easier to
see the information. The important thing is to sort the information by
importance (registration link, then schedule, then logistics (hotel) stuff,
then misc info like the committee) -- and not adding any big images that
would affect load times.

**Comment by IainNZ** [comment]
I'm into the idea of a one page website with a navbar to jump to sections, in the order Leah is describing.
I can take point on this.
How do people feel about the fancy animation at the top? It adds a lot of height...

**Comment by ViralBShah** [comment]
I would prefer a one page website. The committee could be moved to a separate page if too long, but I don't mind that. I would be ok with removing the animation on the top - not sure what others feel.

**Comment by astrieanna** [comment]
I don't care about the animation. Have a clean, fast website that makes it
clear where you are and provides the important information is the important
thing.

**Comment by ArchRobison** [comment]
Single page with navbar works for me.  I don't care about the animation either.

**Comment by IainNZ** [comment]
Done now
